cdk/ingest/lambdas.ts

```diff
@@ -127,7 +124,7 @@ export default class IngestLambdas extends Construct {
     snsTopics.availableDocument.grantPublish(this.scheduler);
 
     this.indexRectifier = new Function(this, 'IndexRectifier', {
-      functionName: `${stage}-obp-cdk-index-rectifier`,
+      functionName: `${stackName}-index-rectifier`,
       handler: 'handler.handler',
```

Comment:
Does this need to be changed to `lambda`?

Reply:
Fixed in c892d65